apps/docs/pages/guides/storage/cdn.mdx

```diff
@@ -66,6 +66,54 @@ If your asset is updated frequently, we recommend that you re-upload the new ass
 
 Objects in public buckets do not require any Authorization to access objects. This leads to a better cache hit rate compared to private buckets. For private buckets, permissions for accessing each object is checked on a per user level. For example, if two different users access the same object in a private bucket from the same region, it results in a cache miss for both the users since they might have different security policies attached to them. On the other hand, if two different users access the same object in a public bucket from the same region, it results in a cache hit for the second user.
 
+## Debugging Cache Hits
+
+The [storage report](https://supabase.com/dashboard/project/_/reports/storage) will give a breakdown of your project's current storage request caching.
+
+A high cache hit rate is desirable to reduce storage egress costs.
+
+The [Logs Explorer](https://supabase.com/dashboard/project/_/logs/explorer) can also be used to debug cache misses.
+
+Cache hits can be determined via the `metadata.response.headers.cf_cache_status` key. Any value corresponding to either `HIT`, `STALE`, `REVALIDATED`, or `UPDATING` would be considered a cache hit.
+The following example query will show the top cache misses from the `edge_logs`:
```

Comment:
What about adding a link to the query like [this](https://supabase.com/dashboard/project/_/logs/explorer?q=select%0A++r.path+as+path%2C%0A++r.search+as+search%2C%0A++count%28id%29+as+count%0Afrom%0A++edge_logs+as+f%0A++cross+join+unnest%28f.metadata%29+as+m%0A++cross+join+unnest%28m.request%29+as+r%0A++cross+join+unnest%28m.response%29+as+res%0A++cross+join+unnest%28res.headers%29+as+h%0Awhere%0A++starts_with%28r.path%2C+%27%2Fstorage%2Fv1%2Fobject%27%29%0A++and+r.method+%3D+%27GET%27%0A++and+h.cf_cache_status+in+%28%27MISS%27%2C+%27NONE%2FUNKNOWN%27%2C+%27EXPIRED%27%2C+%27BYPASS%27%2C+%27DYNAMIC%27%29%0Agroup+by+path%2C+search%0Aorder+by+count+desc%0Alimit+50%3B)